extensions/wikia/PortableInfobox/controllers/ApiQueryPortableInfobox.class.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+
+class ApiQueryPortableInfobox extends ApiQueryGeneratorBase {
+
+	public function __construct( $query, $moduleName ) {
+		parent::__construct( $query, $moduleName, 'ib' );
```

Comment:
ib stand for infobox, this is an params prefix, taking into account we don't use params, maybe we shouldn't worry about that